html2asketch/nodeTreeToSketchGroup.js

```diff
@@ -16,6 +16,22 @@ export default function nodeTreeToSketchGroup(node, options) {
     // Recursively collect child groups for child nodes
     Array.from(node.children)
       .filter(node => isNodeVisible(node))
+      // sort the children by computed z-index so that nodes with lower z-indexes are added
+      // to the group first, "beneath" those with higher z-indexes
+      .sort(function(a, b) {
+        const computedA = getComputedStyle(a).zIndex,
+          computedB = getComputedStyle(b).zIndex,
+          zIndexA = isNaN(computedA) ? 0 : +computedA,
+          zIndexB = isNaN(computedB) ? 0 : +computedB;
+
+        if (zIndexB < zIndexA) {
+          return 1;
+        }
+        if (zIndexA < zIndexB) {
+          return -1;
+        }
+        return 0;
```

Comment:
We can simplify those if's (27-33) like so (arguably it's a bit more readable but it's a widely used pattern):

```suggestion
        return zIndexA - zIndexB;
```